README.md

```diff
@@ -44,65 +41,25 @@ This repository contains the Ansible scripts for installing and configuring WSO2
 │   │   └── tasks
 │   │       ├── custom.yml
 │   │       └── main.yml
-│   ├── is
-│   │   ├── tasks
-│   │   │   ├── custom.yml
-│   │   │   └── main.yml
-│   │   └── templates
-│   │       ├── carbon-home
-│   │       │   ├── bin
-│   │       │   │   └── wso2server.sh.j2
-│   │       │   └── repository
-│   │       │       ├── conf
-│   │       │       │   ├── axis2
-│   │       │       │   │   └── axis2.xml.j2
-│   │       │       │   ├── carbon.xml.j2
-│   │       │       │   ├── datasources
-│   │       │       │   │   ├── bps-datasources.xml.j2
-│   │       │       │   │   ├── master-datasources.xml.j2
-│   │       │       │   │   └── metrics-datasources.xml.j2
-│   │       │       │   ├── identity
-│   │       │       │   │   └── identity.xml.j2
-│   │       │       │   ├── registry.xml.j2
-│   │       │       │   ├── tomcat
-│   │       │       │   │   └── catalina-server.xml.j2
-│   │       │       │   └── user-mgt.xml.j2
-│   │       │       └── deployment
-│   │       │           └── server
-│   │       │               └── eventpublishers
-│   │       │                   ├── IsAnalytics-Publisher-wso2event-AuthenticationData.xml.j2
-│   │       │                   ├── IsAnalytics-Publisher-wso2event-RoleData.xml.j2
-│   │       │                   ├── IsAnalytics-Publisher-wso2event-SessionData.xml.j2
-│   │       │                   └── IsAnalytics-Publisher-wso2event-UserData.xml.j2
-│   │       └── wso2is.service.j2
-│   ├── is-analytics-dashboard
-│   │   ├── tasks
-│   │   │   ├── custom.yml
-│   │   │   └── main.yml
-│   │   └── templates
-│   │       ├── carbon-home
-│   │       │   ├── conf
-│   │       │   │   └── dashboard
-│   │       │   │       └── deployment.yaml.j2
-│   │       │   └── wso2
-│   │       │       └── dashboard
-│   │       │           └── bin
-│   │       │               └── carbon.sh.j2
-│   │       └── wso2is-analytics-dashboard.service.j2
-│   └── is-analytics-worker
-│       ├── tasks
-│       │   ├── custom.yml
-│       │   └── main.yml
-│       └── templates
-│           ├── carbon-home
-│           │   ├── conf
-│           │   │   └── worker
-│           │   │       └── deployment.yaml.j2
-│           │   └── wso2
-│           │       └── worker
-│           │           └── bin
-│           │               └── carbon.sh.j2
-│           └── wso2is-analytics-worker.service.j2
+│   └── is
+│       ├── tasks
+│       │   ├── custom.yml
+│       │   └── main.yml
+│       └── templates
+│           ├── carbon-home
+│           │   ├── bin
+│           │   │   └── wso2server.sh.j2
+│           │   └── repository
+│           │       ├── conf
+│           │       │   └── deployment.toml.j2
+│           │       └── deployment
+│           │           └── server
+│           │               └── eventpublishers
+│           │                   ├── IsAnalytics-Publisher-wso2event-AuthenticationData.xml.j2
```

Comment:
@Savidude We do not need these XML files as analytics is removed. Shall we remove them as well?

Reply:
Removed eventpublishers https://github.com/wso2/ansible-is/pull/45/commits/0dbe8a4f2bdb747caea0f887882b105350769c09